integration-tests/src/test/java/tachyon/client/IsolatedTachyonFileSystemIntegrationTest.java

```diff
@@ -57,10 +58,13 @@ public final void after() throws Exception {
 
   @Before
   public final void before() throws Exception {
-    System.setProperty(Constants.USER_FILE_BUFFER_BYTES, Integer.toString(USER_QUOTA_UNIT_BYTES));
+    MasterContext.getConf().set(Constants.USER_FILE_BUFFER_BYTES, Integer.toString(
```

Comment:
Does changing the master conf affect the client conf? (I'm assuming USER_FILE_BUFFER_BYTES is some client parameter)